apps/src/utils.js

```diff
@@ -708,3 +708,7 @@ export function resetAniGif(element) {
   element.src = '#';
   setTimeout(() => element.src = src, 0);
 }
+
+export function analyticsEvent(category, action, label, value, fieldsObject) {
+  window.ga && window.ga('send', 'event', category, action, label, value, fieldsObject);
```

Comment:
We already have `trackEvent` for this: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/util/trackEvent.js

Reply:
That makes more sense, not sure how I missed it.